www/docs/customization/announce/bluesky.md

```diff
@@ -0,0 +1,29 @@
+# Bluesky
+
+To use [Bluesky](https://bsky.app/), you need to create an account, and set the following
+environment variable on your pipeline:
+
+- `BLUESKY_ACCOUNT_PASSWORD`
```

Comment:
this is an app password, right?

maybe we could clarify that in the previous statement?